DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/add-quartus/system_console/load_vals_functor.tcl

```diff
@@ -0,0 +1,11 @@
+#  Copyright (c) 2022 Intel Corporation                                  
```

Comment:
Same remark: I hope that we can avoid shipping this file

Reply:
see other comment about system_console scripts